ui/src/app/views/sign-in/component.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import {Button} from 'app/components/buttons';
+
+import {mount} from 'enzyme';
+import * as React from 'react';
+
+import SignInReact, {pageImages, SignInProps} from './component';
+
+describe('SignInReact', () => {
+  let props: SignInProps;
+
+  const signIn = jest.fn();
+
+  const component = () => {
+    return mount(<SignInReact {...props}/>);
+  };
+
+  beforeEach(() => {
+    props = {
+      onInit: jest.fn(),
+      signIn: signIn,
+      windowSize: {width: 1400, height: 0}
+    } as SignInProps;
+  });
+
+  it('should display login background image and directive by default', () => {
+    const wrapper = component();
+    const templateImage = wrapper.find('[data-test-id="template"]');
+    const backgroundImage = templateImage.prop('style').backgroundImage;
+    expect(backgroundImage).toBe('url(\'' + pageImages.login.backgroundImgSrc + '\')');
+    expect(wrapper.find('[data-test-id="login"]')).toBeTruthy();
+  });
+
+  it('should display small background image when window width is between 900 and 1300', () => {
+    props.windowSize.width = 999;
+    const wrapper = component();
+    const templateImage = wrapper.find('[data-test-id="template"]');
+    const backgroundImage = templateImage.prop('style').backgroundImage;
+
+    expect(backgroundImage)
+        .toBe('url(\'' + pageImages.login.smallerBackgroundImgSrc + '\')');
+    expect(wrapper.find('[data-test-id="invitation"]')).toBeTruthy();
```

Comment:
What code is this particular assertion trying to exercise? (note: value is incorrect, but obscured by the `exists` issue)